### DISCUSSION TITLE
Build documentation on MOOSE-based App

### DISCUSSION THREAD

**Discussion by chunhuizhao478** [idaholab/moose#26814] [Q&A General]
Hi all, we would like to create and document our dynamic rupture code in our MOOSE-based app, similar as RACOON, I have a question regarding the documentation system after I run `./moosedocs.py generate farmsquakeworxApp`, I encounter the following error:

```
Traceback (most recent call last):
  File "/Users/andyz/projects/farmsquakeworx/doc/./moosedocs.py", line 29, in <module>
    sys.exit(main.run())
  File "/Users/andyz/projects/moose/python/MooseDocs/main.py", line 63, in run
    errno = generate.main(options)
  File "/Users/andyz/projects/moose/python/MooseDocs/commands/generate.py", line 47, in main
    _, _, app_reports = moosesqa.get_sqa_reports(opt.config, app_report=True, doc_report=False, req_report=False)
  File "/Users/andyz/projects/moose/python/moosesqa/get_sqa_reports.py", line 40, in get_sqa_reports
    config = mooseutils.yaml_load(config_file) if isinstance(config_file, str) else config_file
  File "/Users/andyz/projects/moose/python/mooseutils/yaml_load.py", line 112, in yaml_load
    with open(filename, 'r') as fid:
FileNotFoundError: [Errno 2] No such file or directory: 'sqa_reports.yml'
```

I wonder how to resolve the issue, thanks!

**Comment by milljm** [comment]
It sounds like a config file is trying to include another config file that isn't present in your situation. Try to find it, and remove that include line:

```bash
cd ~/projects/farmsquakeworx
git grep "sqa_reports.yml"
```
should return a few places where this file is attempting to be included during the doco build process...

Not 100% sure if this is the right answer or not (like, if you _want_ to include SQA stuff). @cticenhour  would know more about this than I...

**Reply by chunhuizhao478** [reply]
Hi @milljm, thanks for your reply. I manually add a "sqa_reports.yml" in the `~/projects/farmsquakeworx/doc` folder, now it seems to build fine

```
Applications:
    farmsquakeworx:
        exe_directory:     ${ROOT_DIR}
        exe_name:          farmsquakeworx
        app_types:
            - farmsquakeworx
        content_directory: doc/content
```
